src/ansys_sphinx_theme/theme/ansys_sphinx_theme/layout.html

```diff
@@ -3,6 +3,12 @@
 {# Append our custom CSS after the bootstrap stylesheet so we can override where necessary #}
 {%- block css %}
   {{ super() }}
+
+  <!-- CSS for DataTables rendering -->
+  <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+  <link href="https://cdn.datatables.net/v/dt/dt-2.2.0/datatables.min.css" rel="stylesheet">
+  <script src="https://cdn.datatables.net/v/dt/dt-2.2.0/datatables.min.js"></script>
```

Comment:
This solves the JQuery issues for the DataTables. However, we are injecting a lot of CSS and JS just for something we barely use. Let me investigate the usage of the `html_context` and see if we can dynamically load this when required.